examples/imagenet/create_imagenet.sh

```diff
@@ -6,9 +6,18 @@ EXAMPLE=examples/imagenet
 DATA=data/ilsvrc12
 TOOLS=build/tools
 
+# Path to the imagenet train and val image folders 
 TRAIN_DATA_ROOT=/path/to/imagenet/train/
 VAL_DATA_ROOT=/path/to/imagenet/val/
 
+# The encodign that will be used to store the images in the database which 
+# will be used as the network's input. The "" option will not use any encoding 
+# for the images. Run ./build/tools/convert_imageset --help to find about
+# other options. For examples you can use the option jpg where all images will 
+# be encodided as jpg images and they will take less space in the hard disk.
+TRAIN_ENCODING=""
+VAL_ENCODING=""
```

Comment:
Check if it is possible to have a different encoding for the `train` dataset and for the `val` dataset.